Framework/Core/test/test_CCDBFetcher.cxx

```diff
@@ -31,11 +31,11 @@ WorkflowSpec defineDataProcessing(ConfigContext const&)
       AlgorithmSpec{
```

Comment:
@ktf, looks like this test is not compiled anymore, should we remove it? There is almost identical code in  `Framework/TestWorkflows/src/test_CCDBFetcher.cxx`

Reply:
yes